test/unit/general/deviceService-test.js

```diff
@@ -25,9 +25,9 @@
 
 const iotAgentLib = require('../../../lib/fiware-iotagent-lib');
 const utils = require('../../tools/utils');
-const request = utils.request;
```

Comment:
This change is not included in PR https://github.com/telefonicaid/iotagent-node-lib/pull/954

Could you elaborate why is included here?